code/datums/status_effects/debuffs.dm

```diff
@@ -655,3 +728,42 @@
 		to_chat(owner, fake_msg)
 
 	msg_stage++
+
+/datum/status_effect/corrosion_curse
+	id = "corrosion_curse"
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	tick_interval = 100
+
+
+/datum/status_effect/corrosion_curse/on_creation(mob/living/new_owner, ...)
+	to_chat(owner, "<span class='warning'>Your feel your body starting to break apart...</span>")
+	. = ..()
+
+/datum/status_effect/corrosion_curse/tick()
+	. = ..()
+	if(!ishuman(owner))
+		return
+	var/mob/living/carbon/human/H = owner
+	var/chance = rand(0,100)
+	switch(chance)
+		if(0 to 19)
```

Comment:
the only good effect here is vomiting, because it has a hardstun attached to it

all of the other effects will take AGES to kill or even noticeably impair someone

Reply:
it is supposed to be mainly an annoyance, remember it lasts for a few minutes, so the damage stacks quite quickly

Reply:
but the damage is spread out across multiple organs, so it will do literally nothing

Reply:
i will figure something out